src/sass/layouts/_contacts.scss

```diff
@@ -1,6 +1,153 @@
-.our-location{
+.contscts{
+    margin-bottom: 39px;
+    @media screen and(min-width:768px){
+        margin-bottom: 40px;
+    }
+    @media screen and(min-width:1200px){
+        margin-bottom: 120px;
+    }
+}
+.locations{
     display: flex;
+    justify-content: center;
+    align-items: center;
     flex-wrap: wrap;
-
     gap: 20px;
-}
\ No newline at end of file
+    list-style-type: none;
+    padding: 0;
+   @media screen and(min-width:768px){
+       flex-wrap: nowrap;
+       gap: 32px;
+    }
+}
+.location__item{
+    background: var(--modal-bg);
+    box-shadow: 0 8px 30px rgb(212 20 67 / 10%);
+    border-radius: 24px;
+    padding: 40px 32px;
+    flex: 1;
+@media screen and(min-width:768px){
+padding: 40px 24px;
+}
+@media screen and(min-width:1200px){
+ padding: 40px ;
+}
+}
+.location__place{
+    padding: 0 25;
```

Comment:
Pls double-check with the design. 
25px is too much. (Mobile)

Reply:
updated